src/redux/rockets/rocketSlice.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable max-len */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { createSelector } from 'reselect';
+
+const initialState = {
+  rockets: [],
+  status: 'idle',
+  error: null,
+};
+
+const baseUrl = 'https://api.spacexdata.com/v4/rockets';
+
+export const fetchRocketsAsync = createAsyncThunk('rockets/fetchRockets', async () => {
+  // eslint-disable-next-line no-useless-catch
```

Comment:
- [ ] You have done really well in implementing the changes requested by the previous code reviewer, :clap: , However, as she mentioned, `"Please kindly do not disable linters, fixing linters helps us to write a strong and clean code base"` 😁, Please resolve that.